pkgs/applications/video/openshot-qt/default.nix

```diff
@@ -47,6 +54,6 @@ mkDerivationWith python3Packages.buildPythonApplication rec {
     '';
     license = with licenses; gpl3Plus;
     maintainers = with maintainers; [ AndersonTorres ];
-    platforms = with platforms; linux;
+    platforms = with platforms; darwin ++ linux;
```

Comment:
```suggestion
    platforms = platforms.unix;
```

Reply:
I don't think it will work with BSD sound system.

Reply:
@veprbl But there is a FreeBSD port for openshot:

https://www.freshports.org/multimedia/openshot

Reply:
Fair enough. Changed to `unix`.

Reply:
Well, as far as I know we don't have FreeBSD testers yet.
Maybe I need to restart that old project...